templates/partials/answers/checkbox.html

```diff
@@ -1,16 +1,16 @@
 {% from "components/checkboxes/_macro.njk" import onsCheckboxes %}
 
 {% set config = {
-  "legend": question_title,
+  "legend": answer.label,
   "id": answer.id,
   "legendClasses": "u-vh",
```

Comment:
Currently the label isn't being rendered in the multiple answers schema.
![image](https://user-images.githubusercontent.com/1754822/131992637-0c3b7ec2-6085-4868-aaff-c9c3d5510245.png)

The `u-vh` class needs to be removed to fix this. Also `dontWrap` needs to be added for legend to work as expected:
```suggestion
  "id": answer.id,
  "legend": answer.label,
  "dontWrap": not answer.label,
```